test/fixtures/simulated-collaboration/index.js

```diff
@@ -0,0 +1,18 @@
+require('../../../backend/core/db/mongo');
+
+const lib = require('./lib');
```

Comment:
Same here, please remove all the thing relate to sumulated-collaboration. it is used for backend test